tests/test_default_options.py

```diff
@@ -0,0 +1,15 @@
+import os
+import subprocess
+
+import pytest
+
+import mesonpy
+
+
+@pytest.mark.parametrize('args', [[], ['-Dbuildtype=release']], ids=['defaults', '-Dbuildtype=release'])
+def test_ndebug(package_scipy_like, tmp_path, args):
+    with mesonpy._project({'setup-args': args}) as project:
+        command = subprocess.run(
+            ['ninja', '-C', os.fspath(project._build_dir), '-t', 'commands', '../../mypkg/extmod.c^'],
```

Comment:
The `^` is a typo I think?

Reply:
The `^` asks `ninja` to report the build command for the target having the file preceding the `^` as part of its inputs. See the second paragraph here https://ninja-build.org/manual.html#_running_ninja

Reply:
Ah interesting, didn't know that.